scripts/create_AR_DeployInst.py

```diff
@@ -362,9 +365,54 @@ def deploy_without_share(prototype, reactors, reactor_prototypes, power):
         num_rxs = math.ceil(power / reactor_prototypes[prototype][0])
     else:
         num_rxs = math.floor(power / reactor_prototypes[prototype][0])
+    if num_rxs < 0:
+        num_rxs = 0
 
     return num_rxs
 
+def redeploy_reactors(power, prototype, reactor_prototypes, deploy_schedule, item):
+    '''
+    Determines the number of advanced reactors to be redeployed. This function
+    initially assumes that the number of reactors decommissioning at a 
+    given time step (the current timestep minus the lifetime of the 
+    prototype) are to be redeployed. Then the function determines the 
+    number of prototypes needed to fill the current gap in energy demand. 
+    If this calculated number is lower than the number initially deployed, 
+    then the calculated number of the prototype is deployed. 
+
+    The logic behind this function is to maintain a relatively constant 
+    number of each prototype deployed while also minimizing the 
+    oversupply of energy and minimizing the number of reactors deployed. 
+
+    Parameters:
+    -----------
+    power: float
+        gap between energy demand and production at a given time step
+    prototype: str 
+        name of prototype to be deployed 
+    reactor_prototypes: dict
+        information about prototypes, 
+        {name(str):(power(float),lifetime(int))}
+    deploy_schedule: dict
+        deployment schedule of reactor prototypes with the
+        structure for a DeployInst
+    item: int 
+        index of deploy_schedule['DeployInst']['build_times'] in which 
+        prototype was deployed at timestep current minus prototype lifetime
+
+    Returns:
+    --------
+    num_rxs: int
+        number of a prototype to deploy
+    '''
+    num_rxs = deploy_schedule['DeployInst']['n_build']['val'][item]
+    decreased_deployment = math.ceil(power/reactor_prototypes[prototype][0])
```

Comment:
```suggestion
    reactors = reactor_prototypes[prototype]
    decreased_deployment = math.ceil(power/reactors[0])
```
You don't repeat `reactor_prototypes[prototype]` here, but consider changing this for code consistency. Also, I find it nicer to read.